search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,354 @@
+     cmis:creationDate ASC
+    ```
+
+> **Note:** While the CMIS URI data type is not supported, multi-valued properties and multi-valued predicates as defined in the CMIS specification are supported. For example,
+
+```sql
+select * from ext:doc where 'test' = ANY ext:multiValuedStringProperty
+```
+
+## Supported predicates
+
+A predicate specifies a condition that is true or false about a given row or group. The following predicates are supported:
+
+* Comparison predicates, such as `=`, `<>`, `<`, `<=`, `>=`, `>`, `<>`
+  * `IN` predicate
+  * `LIKE` predicate
+
+    > **Note:** Prefixed expressions perform better and should be used where possible.
+
+* `NULL` predicate 
+* Quantified comparison predicate (`= ANY`)
+* Quantified IN predicate (`ANY .... IN (....)`)
+* `IN_FOLDER` predicate function
+
+## Unsupported predicates
+
+The following predicates are not supported:
+
+* TEXT search predicate, such as `CONTAINS()` and `SCORE()` 
+* `IN_TREE()` predicate
+
+## Supported logical operators
+
+The following logical operators are supported:
+
+* `AND` 
+* `NOT`
+* `OR`
+
+## Other operators
+
+In the following cases, the query will go to the database but the result might not be as expected. In all other unsupported cases, the database query will fail and fall back to be executed against the Elasticsearch subsystem.
+
+* `IS NOT NULL`
+* `IS NULL`: Currently, this operator will only find properties that are explicitly NULL as opposed to the property not existing.
+* `SORT`: The multi-valued and `mltext` properties will sort according to one of the values. Ordering is not localized and relies on the database collation. It uses an `INNER JOIN`, which will also filter NULL values from the result set.
+* `d:mltext`: This data type ignores locale. However, if there is more than one locale, the localised values behave as a multi-valued string. Ordering on `mltext` will be undefined as it is effectively multi-valued.
+* `UPPER()` and `LOWER()`: Comparison predicates provide additional support for SQL `UPPER()` and LOWER() functions (that were dropped from a draft version of CMIS specification but are supported for backward compatibility).
+
+## Configuring transactional metadata query
+
+Configure the transaction metadata query using the subsystem properties.
+
+The common properties used to configure the transactional metadata query for the search subsystems are:
+
+* `solr.query.cmis.queryConsistency`
+* `solr.query.fts.queryConsistency`
+
+These properties should be set in the `<TOMCAT_HOME>/shared/classes/alfresco-global.properties` file.
+
+The default value for these properties is `TRANSACTIONAL_IF_POSSIBLE`. However, you can override it with any of the following permitted values:
+
+* `EVENTUAL`
+* `TRANSACTIONAL`
+
+The `solr.query.cmis.queryConsistency` and `solr.query.fts.queryConsistency` properties can also be set per query on the `SearchParameters` and `QueryOptions` objects.
```

Comment:
```suggestion
The `solr.query.cmis.queryConsistency` and `solr.query.fts.queryConsistency` properties can also be set per query on the `SearchParameters` and `QueryOptions` objects in the Java Public API.
```

Reply:
The `solr.query.cmis.queryConsistency` and `solr.query.fts.queryConsistency` properties can also be set per query on the `SearchParameters` and `QueryOptions` objects in the V1 REST API.

This is there. I changed the PUBLIC to be V1 as you mentioned somewhere else. If this isnt correct let me know :-)